crates/bevy_ecs/src/component.rs

```diff
@@ -686,18 +686,24 @@ pub struct ComponentTicks {
 }
 
 impl ComponentTicks {
-    #[inline]
     /// Returns `true` if the component was added after the system last ran.
+    #[inline]
     pub fn is_added(&self, last_run: Tick, this_run: Tick) -> bool {
         self.added.is_newer_than(last_run, this_run)
     }
 
-    #[inline]
     /// Returns `true` if the component was added or mutably dereferenced after the system last ran.
+    #[inline]
     pub fn is_changed(&self, last_run: Tick, this_run: Tick) -> bool {
         self.changed.is_newer_than(last_run, this_run)
     }
 
+    /// Returns the change tick recording the time this data was most recently changed.
+    #[inline]
+    pub fn last_changed(&self) -> u32 {
+        self.changed.get()
+    }
```

Comment:
```suggestion
    pub fn last_changed(&self) -> Tick {
        self.changed
    }
```

All the other methods work on `Tick`, I would prefer this one too

Reply:
The `last_changed` method of [`DetectChanges`](https://docs.rs/bevy/latest/bevy/ecs/change_detection/trait.DetectChanges.html#tymethod.last_changed) returns a `u32` instead of a `Tick` and there is no public method on `Tick` to get the `u32`, so returning the `Tick` instead would not fix the problem of enabling 3rd parties to implement `DetectChanges`.

I suppose an option would be to make the `Tick::get` method `pub`, then have `DetechChanges::last_changed` return `Tick` instead of `u32`. Then, I think `ComponentTick` could be deleted, the `EntityRef::get_change_tick` could then return a plain `Tick`.

Reply:
> there is no public method on `Tick` to get the `u32`

There is? https://dev-docs.bevyengine.org/bevy/ecs/component/struct.Tick.html#method.get

Reply:
yeah my bad. I was using the `0.10` docs. I made the change :+1: